romanisim/tests/test_image.py

```diff
@@ -635,3 +635,109 @@ def test_reference_file_crds_match(level):
     else:
         # level = 2
         assert (type(im) is WfiImage)
+
+
+@metrics_logger("DMS231")
+@pytest.mark.soctests
+def test_inject_source_into_image():
+    """Inject a source into an image.
+    Demonstrates DMS231.
+    """
+
+    # Set constants and metadata
+    galsim.roman.n_pix = 4088
```

Comment:
If we make this much smaller, does it cause problems?  usecrds=False, so it's probably okay?

Reply:
I reduced the size greatly. As part of that, I restricted the simulated objects to PSFs, to better fit the smaller image.